src/hooks/useRepositoriesList.ts

```diff
@@ -0,0 +1,20 @@
+import useSWR from "swr";
+
+interface PaginatedRepoResponse {
+  readonly data: DbRepo[];
+  readonly meta: PageMetaDto;
+}
+
+const useRepositoriesList = (orderBy = "stars", limit = 10) => {
```

Comment:
this is not taking a page argument, just increasing the limit. this makes it such a way that we cannot ever list more than 1000 repos while also, pulling more and more data from the backend; that is such an incremental way of pulling new data that I think it's worth pushing this beta back until everything is ready, since this is a quick fix and there are other inconsistencies 😓

Reply:
https://github.com/open-sauced/hot/issues/388